registry/quilt_server/views.py

```diff
@@ -88,6 +91,13 @@
     aws_secret_access_key=app.config.get('AWS_SECRET_ACCESS_KEY')
 )
 
+sts_client = boto3.client(
+    'sts',
+    region_name='us-east-1',
```

Comment:
We absolutely cannot hard-code the region here. I don't the region is needed at all for this code. @meffij I have a branch `cross-account` that uses `sts` if you want an example.

Reply:
this has been removed